lib/autoyast.rb

```diff
@@ -278,14 +284,38 @@ def apply_extracted_files(scope)
 
     @system_description[scope].files.map do |file|
       if file.user
-        @chroot_scripts << "chown #{file.user}:#{file.group} '#{File.join("/mnt", file.name)}'"
+        @chroot_scripts << "chown --no-dereference #{file.user}:#{file.group} '#{File.join("/mnt", file.name)}'"
```

Comment:
Line is too long. [112/100]<br>Prefer single-quoted strings inside interpolations.